MIP16/mip16.md

```diff
@@ -58,9 +58,16 @@ It is important to note that due to the frequency of such changes, the Weekly Cy
-- Facilitators that have been onboarded into [MIP38: DAO Primitives State](https://github.com/makerdao/mips/blob/master/MIP38/mip38.md) have been authorized by Maker Governance to efficiently interact with the weekly cycle to enable them to fulfill their Core Unit Mandate with as little friction as possible. Facilitators are authorized to submit non-standard Weekly polls that are related to their Core Unit Mandate and include logic in the weekly Executive Vote that is related to their Core Unit Mandate. If necessary, and if it is related to their Core Unit Mandate, Facilitators can skip the Non-Standard Weekly poll and include logic directly into the weekly Executive Vote.
 - To protect the integrity of existing MIPs, Signal Requests are only permitted when they are specifically triggered and required by an existing MIP.
 - This component explicitly forbids the possibility of community-triggered Signal Requests about arbitrary issues. It is also explicitly specified that the only way to make arbitrary changes to Maker Governance is through the full MIPs process or through Governance Mandates of Mandated Actors.
+- Facilitators that have been onboarded into [MIP38: Core Unit MIP Set Definitions](https://github.com/makerdao/mips/blob/master/MIP38/mip38.md) have been authorized by Maker Governance to efficiently interact with the weekly cycle to enable them to fulfill their Core Unit Mandate with as little friction as possible. Facilitators are authorized to submit non-standard Weekly polls that are related to their Core Unit Mandate and include logic in the weekly Executive Vote that is related to their Core Unit Mandate. If necessary, and if it is related to their Core Unit Mandate, Facilitators can skip the Non-Standard Weekly poll and include logic directly into the weekly Executive Votes.
+- Items will not be included in an executive vote until the following requirements have been met, this includes includes items proposed by Core Unit Facilitators:
+   - For code inclusions, smart contract code must be provided to PE-001 by the appropriate Core Unit or counterparty at least two weeks prior to the requested executive vote going live to ensure it can be properly reviewed.
+   - For collateral onboarding, all relevant smart contract addresses and documentation (including legal documentation, where appropriate) must be provided to PE-001 and/or the Governance Facilitators by the relevant counterparties at least two weeks prior to the executive vote going live.
```

Comment:
Here I would enforce that Legal Docs need to be included in the "Onboarding Gov Poll" for the deal, cuz to me, that's the "full deal details": Credit Risk Assessment, Legal Assessment, Technical Assessment and Legal Docs (ruling the legal entities and the real world and the vote makes the documents binding in the real world after 30 days).


There is a counter argument: asking the counterparties to execute final legal docs without knowing if the Onboarding Gov Poll (credit risk, tech and legal assessments) have passed is asking too much to the counter party.

In the ["Legal Process" that I designed with Christian ](https://collateral.makerdao.com/governance-and-collateral-onboarding/rwa-legal-documentation-operations-guide#process-for-managing-legal-documentation-for-collateral-onboarding) he did agree to have legal docs in the Gov Poll stage and the legal docs to be written with wording such as:

> Other than this section and sections [cross-reference confidentiality], [cross-reference governing law], [cross-reference disputes] and [cross-reference miscellaneous] [cross-reference any other sections which the parties may consider appropriate], which shall be binding on the parties upon execution of this Agreement, the other provisions of this Agreement and each other financing agreement shall only become binding and effective on the parties [thirty (30)] days after the affirmative approval of Maker token holders of the transactions contemplated by this Agreement and the other financing agreements, as evidence under the tab “Polling” at vote.makerdao.com

Reply:
So I agree, but that's kind of out of scope for this amendment

Reply:
that should form part of a ratified collateral onboarding framework

Reply:
OK, then this "legal Docs" topics should go into the RWA scope maybe.